infra/Pos/Slotting/Ntp.hs

```diff
@@ -76,61 +63,12 @@ makeLenses ''NtpSlottingState
 ----------------------------------------------------------------------------
 
 -- | Monad transformer which implements NTP-based solution for slotting.
-newtype NtpSlotting m a = NtpSlotting
-    { getNtpSlotting :: ReaderT NtpSlottingVar m a
-    } deriving ( Functor
-               , Applicative
-               , Monad
-               , MonadTrans
-               , MonadIO
-               , MonadFix
-
-               , MonadThrow
-               , MonadCatch
-               , MonadMask
-
-               , MonadBase base
-
-               , HasLoggerName
-               , CanLog
-               , MonadSlotsData
-               )
+type NtpSlotting = Ether.ReaderT NtpSlottingVar
 
-----------------------------------------------------------------------------
--- Common instances used all over the code
-----------------------------------------------------------------------------
+type MonadNtpSlotting = Ether.MonadReader NtpSlottingVar
```

Comment:
This constraint describes a monad that supports the `askNtpSlotting` method. You can see how it was used to improve the code in `src/Pos/Wallet/Web/Server/Full.hs`:

```
-    ntpSlotVar <- lift . lift . lift . lift . lift . lift $ NtpSlotting ask
+    ntpSlotVar <- askNtpSlotting
```